dataverse-webapp/src/main/webapp/notificationTemplate.xhtml

```diff
@@ -0,0 +1,9 @@
+<ui:composition
+        xmlns:ui="http://java.sun.com/jsf/facelets">
+
+
+
+
+
+</ui:composition>
+<!-- Template was created in order to support notification implementations in other repositories.-->
```

Comment:
Maybe comment should say:
`This template is intended to be overridden by Dataverse extensions to support notifications with custom types`